src/ol/object.js

```diff
@@ -163,6 +163,8 @@ ol.ObjectAccessor.prototype.transform = function(from, to) {
  * details, and see {@link ol.dom.Input} for the specific case of binding an
  * object with an HTML element.
  *
+ * Own observable properties may be removed by calling `unset()`.
```

Comment:
I don't think we use the term "own observable property" elsewhere. And I don't really understand it. So I'd just say "Properties can be deleted by using the unset method. E.g. object.unset('foo')."